ansible/roles/ocp-workload-dm7-qlb-demo/tasks/workload.yml

```diff
@@ -98,19 +138,30 @@
     kie_server_route: "{{ oc_get_route_output.stdout }}"
 
 - name: "Configure config file kie-server route"
-  shell: "sed \"s/.*kieserver_host.*/\ \ \ \ \\'kieserver_host\\'\ :\ \\'{{kie_server_route}}\\',/g\" /tmp/{{guid}}/config.js.orig > /tmp/{{guid}}/config.js"
+  shell: >-
+    sed "s/.*kieserver_host.*/    \'kieserver_host\' : \'{{ kie_server_route }}\',/g"
```

Comment:
This is different from original, must be tested.

Reply:
This one does read slightly differently in yaml to json as well it doesn't insert `\ `, it just leaves the four spaces as is. Could not figure out how to get it to look the same using  `>-`operator.

Reply:
Yeah, this is ugly and tricky to assume it will work (or not work). Just needs to be tested and then we will know.